### ISSUE TITLE
sync failure on first onboarding

### ISSUE THREAD

**Issue by sachins-geekyants** [atsign-foundation/at_client_sdk#300]
**Describe the bug**
On fresh install of wavi.ng after clearing previous data , getting sync failure. 
I was using  'adorabledinosaur' atsign.

**To Reproduce**
1. delete previous installed app and clear cached data.
2. re install wavi.ng and onboard (In my case , I was using @ adorabledinosaur)
3. after some time sync onDone callback gets called with status as failure.

**Expected behavior**
status should have been success

**Screenshots**
<img width="715" alt="Screenshot 2021-09-24 at 12 47 38 AM" src="https://user-images.githubusercontent.com/74917871/134624789-b3bc48c8-9c2b-4c1d-bd5c-38b207c0f53e.png">

**Smartphone (please complete the following information):**
 - Android version 10

**Were you using an @‎application when the bug was found?**
  - @ wavi.ng

**Additional context**

**Comment by yahu1031** [comment]
Hey @sachins-geekyants, I can able to reproduce the issue but not the `FormatException`. Can you please recheck the and retry.
Here are my logs
```
I/flutter (16111): Incoming text in home  when app is closed null
I/flutter (16111): INFO|2021-09-28 13:48:03.517266|KeyChainUtil|atsignMap: {}
I/flutter (16111): INFO|2021-09-28 13:48:09.303870|At Onboarding Flutter|Onboarding...!
I/flutter (16111): INFO|2021-09-28 13:48:09.511243|QR Scan|camera status => PermissionStatus.granted
I/flutter (16111): INFO|2021-09-28 13:48:09.511516|QR Scan|storage status is PermissionStatus.granted
W/IInputConnectionWrapper(16111): getSelectedText on inactive InputConnection
W/IInputConnectionWrapper(16111): requestCursorAnchorInfo on inactive InputConnection
4
W/IInputConnectionWrapper(16111): getTextBeforeCursor on inactive InputConnection
D/InputConnectionAdaptor(16111): The input method toggled cursor monitoring on
I/TextInputPlugin(16111): Composing region changed by the framework. Restarting the input method.
D/InputConnectionAdaptor(16111): The input method toggled cursor monitoring on
I/flutter (16111): INFO|2021-09-28 13:48:18.791124|KeyChainUtil|atsignMap: {}
2
I/flutter (16111): outbound finish handler called
D/FilePickerDelegate(16111): Selected type */*
I/FilePickerUtils(16111): Caching from URI: content://com.android.providers.downloads.documents/document/raw%3A%2Fstorage%2Femulated%2F0%2FDownload%2F%40mangotangostable.atKeys
D/FilePickerUtils(16111): File loaded and cached at:/data/user/0/com.atsign.wavi/cache/file_picker/@mangotangostable.atKeys
D/FilePickerDelegate(16111): File path:[com.mr.flutter.plugin.filepicker.FileInfo@2f370bb]
I/flutter (16111): INFO|2021-09-28 13:48:33.327969|KeyChainUtil|atsignMap: {}
2
I/flutter (16111): outbound finish handler called
I/flutter (16111): INFO|2021-09-28 13:48:42.653383|KeyChainUtil|atsignMap: {}
I/flutter (16111): initializing storage
I/flutter (16111): INFO|2021-09-28 13:48:42.775372|CommitLogKeyStore|Keystore initialized successfully
I/flutter (16111): Hive instance has already been initialized. Please call Hive.init only once if possible.
I/flutter (16111): getHiveSecretFromFile no file found
I/flutter (16111): INFO|2021-09-28 13:48:42.831129|HivePersistenceManager|KeyStore initialized successfully
I/flutter (16111): key.key THEME
I/flutter (16111): key.key THEME_COLOR
I/flutter (16111): INFO|2021-09-28 13:48:48.277536|KeyChainUtil|atsignMap: {@mangotangostable: true}
I/flutter (16111): hide called
I/flutter (16111): SEVERE|2021-09-28 13:48:51.098074|AtLookup|Error in remote verb execution Instance of 'UnAuthenticatedException'
I/flutter (16111): error in follows package init : AT0401: Client authentication failed
I/flutter (16111): outbound finish handler called
I/flutter (16111): INFO|2021-09-28 13:48:53.189076|AtLookup|auth success
I/flutter (16111): INFO|2021-09-28 13:48:53.463511|SyncService|Returning the serverCommitId 1944
I/flutter (16111): INFO|2021-09-28 13:48:53.747062|SyncService|Returning the serverCommitId 1944
I/flutter (16111): outbound finish handler called
I/flutter (16111): SEVERE|2021-09-28 13:48:53.820074|AtLookup|Error in remote verb execution Instance of 'UnAuthenticatedException'
I/flutter (16111): error in follows package init : AT0401: Client authentication failed
I/flutter (16111): syncStatus type : Sync status: SyncStatus.success lastSyncedOn: 2021-09-28 08:18:59.107182Z Exception: null, datachanged : true
```

**Comment by yahu1031** [comment]
Logs on Physical device
Redmi Note 8
Android 10
```
Restarted application in 2,197ms.
I/flutter (26722): Incoming images Value in home  is []
I/flutter (26722): Incoming text in home  when app is closed null
I/flutter (26722): INFO|2021-09-28 15:43:45.724109|KeyChainUtil|atsignMap: {}
I/flutter (26722): INFO|2021-09-28 15:43:51.823114|At Onboarding Flutter|Onboarding...!
I/flutter (26722): INFO|2021-09-28 15:43:53.014252|QR Scan|camera status => PermissionStatus.granted
I/flutter (26722): INFO|2021-09-28 15:43:53.015085|QR Scan|storage status is PermissionStatus.denied
I/TextInputPlugin(26722): Composing region changed by the framework. Restarting the input method.
I/flutter (26722): INFO|2021-09-28 15:44:04.360200|KeyChainUtil|atsignMap: {}
I/flutter (26722): FINER|2021-09-28 15:44:04.378007|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:44:08.243315|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): FINER|2021-09-28 15:44:08.266628|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:44:11.478410|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): outbound finish handler called
I/flutter (26722): FINER|2021-09-28 15:44:15.182912|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:44:17.534308|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): FINER|2021-09-28 15:44:17.535625|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:44:20.431077|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): outbound finish handler called
D/FilePickerDelegate(26722): Selected type */*
I/Timeline(26722): Timeline: Activity_launch_request time:1847049205
I/FilePickerUtils(26722): Caching from URI: content://com.android.externalstorage.documents/document/primary%3AWhatsApp%2FMedia%2FWhatsApp%20Documents%2F%40adorabledinosaur_key.atKeys
D/FilePickerUtils(26722): File loaded and cached at:/data/user/0/com.atsign.wavi/cache/file_picker/@adorabledinosaur_key.atKeys
D/FilePickerDelegate(26722): File path:[com.mr.flutter.plugin.filepicker.FileInfo@5c86492]
I/flutter (26722): INFO|2021-09-28 15:44:31.710328|KeyChainUtil|atsignMap: {}
I/flutter (26722): FINER|2021-09-28 15:44:31.711120|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
W/System  (26722): A resource failed to call close.
I/flutter (26722): FINER|2021-09-28 15:44:35.105470|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): FINER|2021-09-28 15:44:35.106705|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:44:38.365066|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): outbound finish handler called
I/flutter (26722): FINER|2021-09-28 15:44:41.906876|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:44:45.699133|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): FINER|2021-09-28 15:44:45.701659|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:44:48.890361|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): outbound finish handler called
I/flutter (26722): INFO|2021-09-28 15:44:51.990011|KeyChainUtil|atsignMap: {}
I/flutter (26722): initializing storage
I/flutter (26722): FINER|2021-09-28 15:44:52.150444|CommitLogKeyStore|last committed sequence: null
I/flutter (26722): INFO|2021-09-28 15:44:52.151129|CommitLogKeyStore|Keystore initialized successfully
I/flutter (26722): FINER|2021-09-28 15:44:52.159925|HivePersistenceManager|AtPersistence.init received storagePath: /storage/emulated/0/Android/data/com.atsign.wavi/files
I/flutter (26722): Hive instance has already been initialized. Please call Hive.init only once if possible.
I/flutter (26722): FINER|2021-09-28 15:44:52.163745|HivePersistenceManager|getHiveSecretFromFile fetching hiveSecretString for @adorabledinosaur from file
I/flutter (26722): FINER|2021-09-28 15:44:52.164779|HivePersistenceManager|getHiveSecretFromFile found filePath: /storage/emulated/0/Android/data/com.atsign.wavi/files/a9d3bbfd52890113c50a1bf33e7b666aef8aaf3a033c1d8aafebd532d5f8b0f8.hash
I/flutter (26722): getHiveSecretFromFile no file found
I/flutter (26722): FINER|2021-09-28 15:44:52.183136|HivePersistenceManager|AtPersistence.openVault received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:44:52.201510|HivePersistenceManager|AtPersistence.openVault opened Hive box:Instance of 'BoxImpl<dynamic>'
I/flutter (26722): INFO|2021-09-28 15:44:52.201963|HivePersistenceManager|KeyStore initialized successfully
I/flutter (26722): FINER|2021-09-28 15:44:52.214977|NotificationServiceImpl|notification init starting monitor
I/flutter (26722): FINER|2021-09-28 15:44:52.524757|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:44:55.607213|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): FINER|2021-09-28 15:44:56.221167|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:45:00.062505|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): FINER|2021-09-28 15:45:02.976476|Monitor|Authenticating the monitor connection: from result:_964a417e-5fb7-497d-8100-08ee243f54d4@adorabledinosaur:a562cb82-1ccc-464f-8143-63a9b97d5c8a
I/flutter (26722): FINER|2021-09-28 15:45:03.172765|Monitor|Authenticating the monitor connection: pkam:FKZ69OBhyt2yORRheEfIc+YN1dfCP1s32kzgSw5uoFCQdrCKHY/Lah42d9yEMO+q5C8rv//NngC92SjeynNmnc9Rnaw+ofWItFgH5zqCvxJu+bB5ffPHcMSR7ldUHZuCkNnzvycEImVRaxxs/3q19vCYbO9oeGtZm1bdP62EcVhSW5JLLkdPSIZRo6ZK8XZ0f1FQuG5Go6U7d/GrFMMhBAQoewbKiLQ2kd1MatHIIXgBkIuJi3Btvpf0/MBYiT7sDmUx7x3NAuRFWUtef6lDEbWw2cqUHou0Fq5RyRwMliR2y61dpAH0fY/AOoMgpjv/hMmyl+UR8qoZaJa1ZvTmMg==
I/flutter (26722): FINER|2021-09-28 15:45:03.590410|Monitor|Monitor connection authentication successful
I/flutter (26722): FINER|2021-09-28 15:45:03.594942|Monitor|monitor started for @adorabledinosaur with last notification time: null
I/flutter (26722): FINER|2021-09-28 15:45:03.613684|NotificationServiceImpl|added regex to listener statsNotification
I/flutter (26722): FINER|2021-09-28 15:45:03.789164|LocalSecondary|calling sync immediate from local secondary
I/flutter (26722): key.key THEME
I/flutter (26722): key.key THEME_COLOR
I/flutter (26722): FINER|2021-09-28 15:45:03.917266|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:45:03.978241|AtClientImpl|signing public data for key:privateaccount
I/flutter (26722): building loader
I/flutter (26722): INFO|2021-09-28 15:45:04.177223|KeyChainUtil|atsignMap: {@adorabledinosaur: true}
I/flutter (26722): FINER|2021-09-28 15:45:04.184230|LocalSecondary|calling sync immediate from local secondary
I/flutter (26722): hide called
I/flutter (26722): FINER|2021-09-28 15:45:04.303125|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:45:04.308360|NotificationServiceImpl|starting monitor through connectivity listener event
I/flutter (26722): FINER|2021-09-28 15:45:04.308833|NotificationServiceImpl|monitor is already started for @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:45:05.328426|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:45:06.861945|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): FINER|2021-09-28 15:45:06.864216|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): FINER|2021-09-28 15:45:08.139135|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): FINER|2021-09-28 15:45:10.708|AtLookup|from result:data:_a98ebb01-abfa-45b3-97c5-6aca142f5d8f@adorabledinosaur:1b5b2b0f-21b1-41f9-a837-27f8f287cf22
I/flutter (26722): FINER|2021-09-28 15:45:10.712887|AtLookup|fromResponse _a98ebb01-abfa-45b3-97c5-6aca142f5d8f@adorabledinosaur:1b5b2b0f-21b1-41f9-a837-27f8f287cf22
I/flutter (26722): FINER|2021-09-28 15:45:10.807193|AtLookup|Sending command pkam:USDln2jc0G+RRaR+VFGjJIJBoub45xNO3dMHHiKEMxPaD69VPeXJQ1H4M0pupZuzh5xzm1lV5A3vuFz96yX2SihVdtYH5A3Jm99hQlNX0btvSngWY8wzLOCvGfSnPniVEpehUjRz1m+8b0Wxf3VCyX6sg7y23B8Q5WO3N2aDogF4or9W1/tZAwQcGaQzUY+fvxn4bT0CwsqwiAByJxiqQM6ZuLhvnokCuxdxnbbcBltEZzI7PuvDUINpDWhQaZNqaxQQrrMRkKj4ZO5lhLSYIQr3PfmSkE/SU4Ky6kOf5VReOh75O4K/sb9A90+b7nAOLCR5/TpJTuu6WEJ4wq2sRg==
I/flutter (26722): FINER|2021-09-28 15:45:10.809129|AtLookup|from result:data:_7b3820fa-bbe1-4a8d-8d63-95741e9cb383@adorabledinosaur:a0958109-4402-4e84-ac3b-f186c3f5d263
I/flutter (26722): FINER|2021-09-28 15:45:10.810558|AtLookup|fromResponse _7b3820fa-bbe1-4a8d-8d63-95741e9cb383@adorabledinosaur:a0958109-4402-4e84-ac3b-f186c3f5d263
I/flutter (26722): FINER|2021-09-28 15:45:10.838025|AtLookup|Sending command pkam:iXxWFJE6Sd6nvTQCyviGOrwfWy0ZfC2qnotixhCGusRMJQ/WmjpZyEAuMZK4+NK+XZpCl/qcKYOTHdDfqA7GQUHG2CfUnlyaE1OdUZF8hXAmdRJp8XOq4WPtfXVstjP0djYLmy6ULp0OeNNmTVzn8h5EtXUnX8Ybcq1sKFCKVXm0uLfeE/Cu/1h+5hI82o6E01NAYwhtgN5Ju0ZVOfoIBcn3s3495i+IhkuA/ThphXJH/iLgqUks/AEv9b/MzjBBQKU01TR7g/VVcbJYygFqhZS8BsSsNTz8EqDt1PJcAadvAqxvfkFjbqtnSUFJxQjCPtZq91+XdZirX+JPYHVHug==
I/flutter (26722): outbound finish handler called
I/flutter (26722): SEVERE|2021-09-28 15:45:12.034065|AtLookup|Error in remote verb execution Instance of 'UnAuthenticatedException'
I/flutter (26722): INFO|2021-09-28 15:45:12.041004|AtLookup|auth success
I/flutter (26722): FINER|2021-09-28 15:45:12.042907|AtLookup|AtLookup.findSecondary received atsign: @adorabledinosaur
I/flutter (26722): error in follows package init : AT0401: Client authentication failed
I/flutter (26722): FINER|2021-09-28 15:45:12.615280|AtLookup|from result:data:_344d9c2d-010d-440d-8ff8-9fcc7f8b7633@adorabledinosaur:2dbaa24e-0f5e-45a3-918e-14f19a939abe
I/flutter (26722): FINER|2021-09-28 15:45:12.616756|AtLookup|fromResponse _344d9c2d-010d-440d-8ff8-9fcc7f8b7633@adorabledinosaur:2dbaa24e-0f5e-45a3-918e-14f19a939abe
I/flutter (26722): FINER|2021-09-28 15:45:12.655533|AtLookup|Sending command pkam:kgBZcFEtsO/lBZDqpeIOWhc46SUUA28aPcUVMkeTQqHCqhPuRd03OohMIXbYNyQiXKLyuD45RC+XpGls0AW8Rcsi632hZafxfIATNbRXoTP6SxpOYgnt1gNS/N4rWEw3WjGC5BoAFAO1b3tYMBdZe1tuXw5znSp2J+zmXZondri6WA4DoCsFa8+fzM8RbyXTROgqsabHD/I++xNja4NdEHHSyzJIJ3kpnSeCcINmxZPGgeO3zEAqWwQ0cK4NdXG4ZrE9d6UH0Jd/6fHD8r9Cc1v9DbqT5Ku79mmH8g7Ctge65pKNMjsOYtc0ndog+nGBoh6KJUGBeZz0Xw5ZBmjy9Q==
I/flutter (26722): INFO|2021-09-28 15:45:15.088340|AtLookup|auth success
I/flutter (26722): FINER|2021-09-28 15:45:15.140559|AtLookup|AtLookup.findSecondary got answer: 5e4834ea-dd0c-5a23-b441-7be000cef4fd.hornet.atsign.zone:1936 and closing connection
I/flutter (26722): INFO|2021-09-28 15:45:16.728833|SyncService|Returning the serverCommitId 11585
I/flutter (26722): FINER|2021-09-28 15:45:16.794081|SyncUtil|localCommitId:null
I/flutter (26722): FINER|2021-09-28 15:45:16.795194|SyncUtil|serverCommitId:11585
I/flutter (26722): FINER|2021-09-28 15:45:16.796196|SyncUtil|changed entries: 2
I/flutter (26722): INFO|2021-09-28 15:45:18.646794|SyncService|Returning the serverCommitId 11585
I/flutter (26722): FINER|2021-09-28 15:45:18.655880|SyncService|Sync in progress
I/flutter (26722): FINER|2021-09-28 15:45:18.663906|SyncService|syncing to local
I/flutter (26722): FINER|2021-09-28 15:45:18.678134|SyncService|** syncBuilder sync:from:-1:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:45:19.281596|NotificationServiceImpl|added regex to listener
I/flutter (26722): FINER|2021-09-28 15:45:23.665290|SyncService|** syncBuilder sync:from:32:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:45:25.807047|SyncService|** syncBuilder sync:from:51:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:45:35.954736|SyncService|got stats notification in sync: 11586
I/flutter (26722): INFO|2021-09-28 15:45:35.964726|Connections Service|Received notification:: id:-1 key:statsNotification.@adorabledinosaur operation:update from:@adorabledinosaur to:@adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:45:36.045264|SyncService|** syncBuilder sync:from:189:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:45:58.667180|SyncService|got stats notification in sync: 11586
I/flutter (26722): INFO|2021-09-28 15:45:58.668547|Connections Service|Received notification:: id:-1 key:statsNotification.@adorabledinosaur operation:update from:@adorabledinosaur to:@adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:45:59.741903|SyncService|** syncBuilder sync:from:337:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:46:15.627555|SyncService|got stats notification in sync: 11586
I/flutter (26722): INFO|2021-09-28 15:46:15.629023|Connections Service|Received notification:: id:-1 key:statsNotification.@adorabledinosaur operation:update from:@adorabledinosaur to:@adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:46:15.746293|SyncService|** syncBuilder sync:from:1396:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:46:21.271703|SyncService|** syncBuilder sync:from:1496:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:46:29.594255|SyncService|** syncBuilder sync:from:1573:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:46:30.958594|SyncService|got stats notification in sync: 11586
I/flutter (26722): INFO|2021-09-28 15:46:30.961231|Connections Service|Received notification:: id:-1 key:statsNotification.@adorabledinosaur operation:update from:@adorabledinosaur to:@adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:46:31.096861|SyncService|** syncBuilder sync:from:1583:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:46:33.355578|SyncService|** syncBuilder sync:from:1630:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:46:43.310657|SyncService|** syncBuilder sync:from:1946:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:47:04.252369|SyncService|got stats notification in sync: 11586
I/flutter (26722): INFO|2021-09-28 15:47:04.253864|Connections Service|Received notification:: id:-1 key:statsNotification.@adorabledinosaur operation:update from:@adorabledinosaur to:@adorabledinosaur
I/flutter (26722): FINER|2021-09-28 15:47:04.375745|SyncService|** syncBuilder sync:from:2314:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:47:06.686896|SyncService|** syncBuilder sync:from:2335:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:47:10.715336|SyncService|** syncBuilder sync:from:2534:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:47:12.440527|SyncService|** syncBuilder sync:from:2558:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
I/flutter (26722): FINER|2021-09-28 15:47:14.619897|SyncService|** syncBuilder sync:from:2571:limit:10:(.wavi|atconnections|[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})
I/flutter (26722):
```

**Comment by sachins-geekyants** [comment]
@yahu1031 what was the sync status on physical device ?

**Comment by yahu1031** [comment]
It was success

**Comment by yahu1031** [comment]
Any other updates on this issue @sachins-geekyants?

**Comment by sachins-geekyants** [comment]
today I was running at_setting , I noticed sync failure here as well.
```
flutter: notification received in events package ===========> AtNotification{id: -1, key: statsNotification.@adorabledinosaur, from: @adorabledinosaur, to: @adorabledinosaur, epochMillis: 1633604347013, value: 15321, operation: update}
flutter: returned from _notificationCallback in events package ===========>
flutter: _notificationCallback notification received in location package ===========> :AtNotification{id: -1, key: statsNotification.@adorabledinosaur, from: @adorabledinosaur, to: @adorabledinosaur, epochMillis: 1633604347013, value: 15321, operation: update} , notification key: statsNotification.@adorabledinosaur
flutter: returned from _notificationCallback in location package ===========>
flutter: _notificationCallback notification received in location package ===========> :AtNotification{id: -1, key: statsNotification.@adorabledinosaur, from: @adorabledinosaur, to: @adorabledinosaur, epochMillis: 1633604347013, value: 15321, operation: update} , notification key: statsNotification.@adorabledinosaur
flutter: returned from _notificationCallback in location package ===========>
flutter: INFO|2021-10-07 16:29:07.654090|KeyChainUtil|atsignMap: {}
flutter: INFO|2021-10-07 16:29:11.286119|SyncService|Returning the serverCommitId 15298
flutter: FINER|2021-10-07 16:29:11.309071|SyncUtil|localCommitId:15298
flutter: FINER|2021-10-07 16:29:11.309625|SyncUtil|serverCommitId:15298
flutter: FINER|2021-10-07 16:29:11.310101|SyncUtil|changed entries: 0
flutter: FINER|2021-10-07 16:29:11.310617|SyncService|server and local are in sync - aaa81cde-d4e9-487b-8aa1-22ad8e96a742
flutter: syncStatus : Sync status: SyncStatus.success lastSyncedOn: 2021-10-07 10:59:11.311134Z Exception: null, data changed : false
flutter: FINER|2021-10-07 16:29:11.311593|SyncService|clearing sync queue
flutter: outbound finish handler called
flutter: outbound finish handler called
flutter: INFO|2021-10-07 16:29:22.886072|At Onboarding Flutter|Onboarding...!
flutter: INFO|2021-10-07 16:29:22.926104|KeyChainUtil|atsignMap: {}
flutter: SEVERE|2021-10-07 16:29:22.926571|AtClientService|Atsign not found
flutter: INFO|2021-10-07 16:29:22.948648|QR Scan|camera status => PermissionStatus.granted
flutter: INFO|2021-10-07 16:29:22.948929|QR Scan|storage status is PermissionStatus.granted
flutter: INFO|2021-10-07 16:29:37.652286|KeyChainUtil|atsignMap: {}
flutter: outbound finish handler called
flutter: outbound finish handler called
flutter: INFO|2021-10-07 16:30:06.133654|KeyChainUtil|atsignMap: {}
flutter: outbound finish handler called
flutter: outbound finish handler called
flutter: INFO|2021-10-07 16:30:21.896404|KeyChainUtil|atsignMap: {}
flutter: initializing storage
flutter: INFO|2021-10-07 16:30:22.000388|CommitLogKeyStore|Keystore initialized successfully
flutter: Hive instance has already been initialized. Please call Hive.init only once if possible.
flutter: getHiveSecretFromFile no file found
flutter: INFO|2021-10-07 16:30:22.028556|HivePersistenceManager|KeyStore initialized successfully
flutter: INFO|2021-10-07 16:30:32.508622|AtMeService|Monitor started for null
flutter: INFO|2021-10-07 16:30:32.571|KeyChainUtil|atsignMap: {@adorabledinosaur: true}
flutter: INFO|2021-10-07 16:30:32.583027|DashboardWidget|loaded successfully
flutter: SEVERE|2021-10-07 16:30:35.500126|DashboardWidget|Edit Clicked _getPublicFielfsfsds  false
flutter: INFO|2021-10-07 16:30:39.974156|AtLookup|auth success
flutter: INFO|2021-10-07 16:30:41.107829|SyncService|Returning the serverCommitId 15321
flutter: INFO|2021-10-07 16:30:41.829485|SyncService|Returning the serverCommitId 15321
flutter: _onSuccessCallback 2021-10-07 16:33:34.436856Sync status: SyncStatus.failure lastSyncedOn: 2021-10-07 11:03:34.435784Z Exception: AT0025: FormatException: Unexpected end of input (at character 1)

^
 true
flutter: INFO|2021-10-07 16:33:34.492382|AtMeService|fetched value [255, 216, 255, 224, 0, 16, 74, 70, 73, 70, 0, 1, 1, 0, 0, 72, 0, 72, 0, 0, 255, 225, 0, 88, 69, 120, 105, 102, 0, 0, 77, 77, 0, 42, 0, 0, 0, 8, 0, 2, 1, 18, 0, 3, 0, 0, 0, 1, 0, 1, 0, 0, 135, 105, 0, 4, 0, 0, 0, 1, 0, 0, 0, 38, 0, 0, 0, 0, 0, 3, 160, 1, 0, 3, 0, 0, 0, 1, 0, 1, 0, 0, 160, 2, 0, 4, 0, 0, 0, 1, 0, 0, 1, 144, 160, 3, 0, 4, 0, 0, 0, 1, 0, 0, 1, 9, 0, 0, 0, 0, 255, 237, 0, 56, 80, 104, 111, 116, 111, 115, 104, 111, 112, 32, 51, 46, 48, 0, 56, 66, 73, 77, 4, 4, 0, 0, 0, 0, 0, 0, 56, 66, 73, 77, 4, 37, 0, 0, 0, 0, 0, 16, 212, 29, 140, 217, 143, 0, 178, 4, 233, 128, 9, 152, 236, 248, 66, 126, 255, 192, 0, 17, 8, 1, 9, 1, 144, 3, 1, 34, 0, 2, 17, 1, 3, 17, 1, 255, 196, 0, 31, 0, 0, 1, 5, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 255, 196, 0, 181, 16, 0, 2, 1, 3, 3, 2, 4, 3, 5, 5, 4, 4, 0, 0, 1, 125, 1, 2, 3, 0, 4, 17, 5, 18, 33, 49, 65, 6, 19, 81, 97, 7, 34, 113, 20, 50, 129, 145, 161, <…>
flutter: INFO|2021-10-07 16:33:34.503895|AtMeService|fetched value light for key theme
flutter: SEVERE|2021-10-07 16:33:34.505359|AtMeService|setting the theme key for user throws Invalid argument(s): propery not found
flutter: INFO|2021-10-07 16:33:34.506266|AtMeService|fetched value 3fc0f3 for key theme_color
flutter: SEVERE|2021-10-07 16:33:34.506621|AtMeService|setting the theme_color key for user throws Invalid argument(s): propery not found
flutter: INFO|2021-10-07 16:33:34.506978|DashboardWidget|loaded successfully
flutter: INFO|2021-10-07 16:33:42.054196|SyncService|Returning the serverCommitId 15321
flutter: INFO|2021-10-07 16:33:42.600269|SyncService|Returning the serverCommitId 15321
flutter: _onSuccessCallback 2021-10-07 16:39:13.884719Sync status: SyncStatus.failure lastSyncedOn: 2021-10-07 11:09:13.884671Z Exception: AT0025: FormatException: Unexpected end of input (at character 1)

^
 true

```

**Comment by yahu1031** [comment]
Here are some logs when I used `@adorabledinosaur` in @\rrive app. I have mentioned somewhere **refer from here** please make sure analyze logs about this atsign from there.

**LOGS**
[arrive (1).log](https://github.com/atsign-foundation/at_client_sdk/files/7312168/arrive.1.log)

**Comment by yahu1031** [comment]
As I have mentioned in above comments I can only reproduce

```
I/flutter (26722): SEVERE|2021-09-28 15:45:12.034065|AtLookup|Error in remote verb execution Instance of 'UnAuthenticatedException'
```
Only this but not `FormatException`.

**Comment by sachins-geekyants** [comment]
@yahu1031 in the arrive logs that you have attached , I can see formatException happening with sync failure.

**Comment by yahu1031** [comment]
Yes in arrive it happens and not very sure it is because of what. We already have one issue opened.

**Comment by yahu1031** [comment]
@sachins-geekyants I have reproduced the issue today and recorded the session too. I'm attaching them here. In the logs, I can't see any sync failure issues.

Logs: https://cdn.sourceb.in/bins/xhfj8DYybu/0

- Found an issue, I will raise another issue for that.

**Comment by sachins-geekyants** [comment]
today I was running at_settings  with @ adorabledinosaur atsign , I got these exceptions in sync.
Even after these exceptions , sync was running and I got data in some time.
 

![Screenshot 2021-10-20 at 7 35 42 PM](https://user-images.githubusercontent.com/74917871/138109393-b8fb9819-7292-4acb-a02f-97ca4346d32a.png)
.

**Comment by yahu1031** [comment]
@sachins-geekyants Can you just please let us know if is also happening with the other atsigns...
I have tried with `@minnu💙` atsign

**Comment by sachins-geekyants** [comment]
@yahu1031 I faced it with several atsigns.
like @ minorgettingplayed7 , @ 25antwilling , @ adorabledinosaur

**Comment by yahu1031** [comment]
@sachins-geekyants I too faced the `FormatException` today. Unfortunately didn't save the logs. Will try again and attach the logs if I can reproduce them. Can I please get a key for any of the above atsigns you have mentioned? I will try with them too from my end.

**Comment by sachins-geekyants** [comment]
@yahu1031 sure , I will send you.

**Comment by yahu1031** [comment]
@sachins-geekyants I have tried reproducing this with `@minorgettingplayed7`

Here are the [logs](https://cdn.sourceb.in/bins/akvwwZvq1f/0)

I got sync success but in middle, I found this. I'm checking this

```
I/flutter (19950): SEVERE|2021-11-08 19:21:11.287113|AtLookup|Error in remote verb execution Instance of 'UnAuthenticatedException'
I/flutter (19950): error in follows package init : AT0401: Client authentication failed
I/flutter (19950): SEVERE|2021-11-08 19:21:55.961849|AtLookup|Error in remote verb execution Instance of 'UnAuthenticatedException'
I/flutter (19950): error in follows package init : AT0401: Client authentication failed
```

**Comment by sachins-geekyants** [comment]
@yahu1031 I didn't get 'UnAuthenticatedException' , tried reproducing it.

**Comment by yahu1031** [comment]
@sachins-geekyants So the app runs smooth? 
Like please LMK if still we have any expected and actual behavior. As of now till we get this issue again I will close this. And feel free to open it again if something goes wrong.

**Comment by sachins-geekyants** [comment]
In my testing with @ minorgettingplayed7 on wavi.ng, app was working smoothly.
We need to test with other apps and atsigns as well, If this issue comes up again we can re open the ticket.
